json-examples/slice-provisionning.json

```diff
@@ -0,0 +1,137 @@
+{
+  "ietf-network-slice-service:network-slice-services": {
+    "slo-sle-templates": {
+      "slo-sle-template": [
+        {
+          "id": "low-latency-template",
+          "template-description": "lowest possible latencey forwarding behavior"
+        }
+      ]
+    },
+    "slice-service": [
+      {
+        "service-id": "Slice EMBB_UP",
+        "service-description": "Dedicate TN Slice for EMBB-UP",
+        "slo-sle-template": "low-latency-template",
+        "status": {},
+        "sdps": {
+          "sdp": [
+            {
+              "sdp-id": "sdp1",
+              "ac-ref": [
+                {
+                  "name":"ac1"
+                }
+              ]
+            },
+            {
+              "sdp-id": "sdp2",
+              "ac-ref": [
+                {
+                  "name":"ac2"
+                }
+              ]
+            }
+          ]
+        }
+      }
+    ]
+  },
+  "ietf-ac-svc:attachment-circuits": {
+    "ac": [
+      {
+        "name": "ac1",
+        "description": "Connection to site1 on vlan 100 for slice EMBB_UP",
+        "requested-ac-start": "2023-12-12T05:00:00.00Z",
+        "l2-connection": {
+          "encapsulation": {
+            "type": "ietf-vpn-common:dot1q",
+            "dot1q": {
+              "tag-type": "ietf-vpn-common:c-vlan",
+              "cvlan-id": 100
+            }
+          },
+          "bearer-reference": "bearerX@site1"
+        },
+        "ip-connection": {
+          "ipv4": {
+            "local-address": "172.30.1.2",
+            "prefix-length": 30,
+            "address": [
+              {
+                "address-id": "1",
+                "customer-address": "172.30.1.1"
+              }
+            ]
+          }
+        },
+        "routing-protocols": {
+          "routing-protocol": [
+            {
+              "id": "1",
+              "type": "??????????? makes sense or not ?",
```

Comment:
This can be useful to avoid misconfiguration sanity checks and also to ease identifying which network-derived config to put in place.